src/components/DaySelector.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import styles from '../styles';
+
+type DaySelectorProps = {
+  daysOfWeek: { short: string; full: string }[];
+  selectedDay: string | null;
+  onSelectDay: (day: string) => void;
```

Comment:
all of these shouldn't be `string` but rather `enum`s

Reply:
okey dokey